docs/src/howto-guides/002_installation.md

```diff
@@ -10,13 +10,56 @@ curl -fsSL https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/get-thi
 
 You can execute that command on your device and it will do all required steps for an initial setup.
 
-If you prefer to have a little more control over the installation or the script did not work for you, please go on with the following steps.
+> Note: If you want to get a specific version, add `<version>` at the end like below. 
+> `<version>` is consist of 3 digits, e.g. `0.7.3`.
+> ```shell
+> curl -fsSL https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/get-thin-edge_io.sh | sudo sh -s <version>
+> ```
```

Comment:
Why is this a quote?

Reply:
For this, because it's A or B relation. User will run either the command without or with version. I believe the main stream is to use the first command, which is without version specified. That's why I thought it would be better keeping with version inside "Note".

But I don't have so strong opinion. If it looks too weird, then I can change.

Reply:
If you have a look at the rich diff, it does indeed look a bit weird, because it looks like you're quoting someone... which of course you don't do... :smile:

Reply:
But it looks good on the mdbook :(

Reply:
Now I don't know what you want... 

If your reason is "the quote block looks quoting someone's saying", I disagree. It's meant for writing "Note", and it's commonly used for that purpose. 

If your reason is "the quote block should be promoted to normal text block as it does not look really 'Note'", then I will consider to change. At least, the big quote block under upgrading section.

Reply:
Hm, okay. :shrug: :+1: